_posts/2015-11-16-behat-to-the-fixture.md

```diff
@@ -210,7 +210,7 @@ class FixturesContext implements Context
 
 {% endhighlight %}
 
-First, check the database host is *localhost* (on a vagrant for example), to avoid to clear wrong databases.
+First, ensure that the database host is *localhost* (on a vagrant for example), in order to avoid to clearing wrong databases.
```

Comment:
As my first review, I would say `to avoid clearing`, not `to avoir to clearing`.